verif/big_core/tests/dino_game.c

```diff
@@ -1,27 +1,28 @@
-#include "big_core_defines.h"
-#include "graphic_vga.h"
+#include "../../../app/defines/big_core_defines.h"
```

Comment:
There should be no need for full path.
Need to add the search path in the build